abc.js

```diff
@@ -183,18 +183,18 @@ email.addEventListener('input', () => {
   }
 });
 form.addEventListener('submit', (event) => {
-  event.preventDefault();
   const valid = email.value.length === 0 || emailRegExp.test(email.value);
   if (!valid) {
     email.className = 'invalid';
     error.textContent = 'Form is not sent Type email in lowercase please and try again';
     error.className = 'error active';
+    event.preventDefault();
   } else {
     email.className = 'valid';
     error.textContent = 'The form has been submitted';
     error.className = 'delievered';
   }
-  body.addEventListener('click', ()=> {
+  body.addEventListener('click', () => {
     error.textContent = '';
```

Comment:
- [ ] Kindly make sure your form can be submitted when the user clicks the `Get in touch` button.
- [ ] Also consider providing some error message if a wrong inputs email was put validating the users inputs.

Reply:
it can ben submitted when the user clicks get in touch
now the user also getting an error it he is putting a wrong input in an email section